tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java

```diff
@@ -82,6 +82,12 @@
     return list;
   }
 
+  /**
+   * Constructor for {@RemoteBlockInStreamIntegrationTest}.
```

Comment:
I think this needs to be {@link RemoteBlockInStreamIntegrationTest} for the javadoc to create the link correctly